src/dtx/dtx_rpc.c

```diff
@@ -369,37 +369,27 @@ struct dtx_common_args {
 
 /* If is_reentrance, this function ignores len. */
 static int
-dtx_req_list_send(struct dtx_common_args *dca, daos_epoch_t epoch, int len, bool is_reentrance)
+dtx_req_list_send(struct dtx_common_args *dca, bool is_reentrance)
```

Comment:
[Not requesting changes] Should the comment on new line 370 needs to be remove?

Reply:
OK, I will update the comment.